docs/sources/flow/reference/components/otelcol.processor.k8sattributes.md

```diff
@@ -225,6 +225,8 @@ Name | Type     | Description
 
 The `exclude` block configures which pods to exclude from the processor.
 
+> **Note**: Pods with name `jaeger-agent` and `jaeger-collector` are excluded by default.
```

Comment:
```suggestion
{{< admonition type="note" >}}
Pods with the name `jaeger-agent` or `jaeger-collector` are excluded by default.
{{< /admonition >}}
```

Reply:
I don't see this suggestion applied.  Notes should not be formatted as a block quote. They are not correctly styled in the generated web page unless we use the `admonition` shortcode.